### PR TITLE
Add an "experimental feature" note in dashboard-intro.md

### PR DESCRIPTION
**Pull request by yikeke**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/2832#discussion_r440598224
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
PTAL @breeswish

File: dashboard/dashboard-intro.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ category: how-to
 
 # TiDB Dashboard 介绍
 
+> **注意：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。
```
Comment (breeswish) [inline dashboard/dashboard-intro.md:10]: However actually we **recommends** to use in production 🤔

Comment (jackysp) [reply dashboard/dashboard-intro.md:10]: The feedback from the QA team is that it is better to wait for them to test it and then remove this warning.

Comment (yikeke) [reply dashboard/dashboard-intro.md:10]: So is it okay to merge this for now? @breeswish

**Pull request merged**